### ISSUE TITLE
FL 20.6+ Automation clips on tempo events cause System.ArgumentOutOfRangeException when parsing automation clips

### ISSUE THREAD

**Issue by Kaydax** [monadgroup/FLParser#13]
So I made an issue before where I was confused as to why I was getting an Index Out Of Range Exception when trying to parse an flp. Turns out after testing with multiple different automation clips, it is caused by using automation clips on the tempo

**Comment by demberto** [comment]
Hey, @Kaydax I too encountered the same issue, can you confirm that it is caused at this line?
`TargetChannel = _project.Channels[paramDestination],`
where `paramDestination` is 16384.

**Comment by Kaydax** [comment]
Yes @demberto it is indeed that line, and that is the value i also get. Right now i have a basic fix on my fork in my net-standard branch, but I'm very sure it's super hacky and only for my usecase
